test/functional/wallet_balance.py

```diff
@@ -246,7 +257,7 @@ def test_balances(*, fee_node_1=0):
         self.nodes[1].sendrawtransaction(hexstring=tx_replace, maxfeerate=0)
 
         # Now confirm tx_replace
-        block_reorg = self.nodes[1].generatetoaddress(1, ADDRESS_WATCHONLY)[0]
```

Comment:
In commit "Move import and watchonly tests to be legacy wallet only in wallet_balance.py" (8ee94451ad55b81a53c38c6b8cee0c13f1b1f9e7)

I don't get these changes. Does the address generated to matter for the tests or is just code cleanup? It would be good to mention these changes in the commit description and say what they are

Reply:
Reverted.